gubernator/filters.py

```diff
@@ -72,7 +72,7 @@ def do_testcmd(name):
             return name
         return 'go test -v %s -run %s$' % (pkg, name)
     else:
-        name = name.replace('[k8s.io] ', '')
```

Comment:
It does, but I only want it to remove the (repetitive) one at the beginning of the line -- removing it in the middle of the test name makes the regex not match.